cartridges/int_yotpo_sfra/cartridge/templates/default/loyalty/yotpoFooter.isml

```diff
@@ -7,7 +7,8 @@
         this.addEventListener('load', function(c) {
             // this event listener will hit every time an ajax request is made, so filter out
             // requests so we only re-init the widgets when the request is for PLP search refinements
-            if (c.currentTarget.responseURL.includes('Search-ShowAjax')) {
+            var urlObj = new URL(c.currentTarget.responseURL)
```

Comment:
Fine as we should always have a responseURL. Otherwise, newing-up a URL via a `URL(null)` or `URL(undefined)` yields an error.